pkg/components/yql_agent.go

```diff
@@ -141,8 +141,10 @@ func (yqla *yqlAgent) doSync(ctx context.Context, dry bool) (ComponentStatus, er
 
 	if yqla.NeedSync() {
 		if !dry {
+			token, _ := yqla.secret.GetValue(consts.TokenSecretKey)
 			ss := yqla.server.buildStatefulSet()
 			container := &ss.Spec.Template.Spec.Containers[0]
+			container.Command = []string{"sh", "-c", fmt.Sprintf("echo -n %s > /usr/yql_agent_token; %s", token, strings.Join(container.Command, " "))}
```

Comment:
Shouldn't we use `YTTokenPath` instead of hardcoded `/usr/yql_agent_token` here?

Reply:
It is not very easy to access the config here. It has already been serialized by this point. So I put token path in consts